pydis_site/apps/content/resources/guides/python-guides/subclassing_bot.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Subclassing Bot
+description: "Subclassing the Bot to add more functionality and customisability."
+---
+
+## Basic Subclassing
+First,  a [basic article](https://www.codesdope.com/course/python-subclass-of-a-class/) on subclassing will provide some fundamental knowledge, which is highly suggested before moving on to this topic, as subclassing [Bot](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Bot) can ultimately be a complicated task.
+
+## The benefits of subclassing bot
+Subclassing Bot can be very beneficial as it provides you with more control and customisability of how your bot functions, also allowing you to add extra features, such as custom bot attributes or methods. For example, the default [Context](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html#discord.ext.commands.Context) can be overriden to add more functionality.
+
+There are two ways to subclass `commands.Bot`, as shown below:
```

Comment:
I don't think I understand this, you subclass them the same way?

I think what you're talking about here is which "style" it is (depending on how you call `super().__init__()`).

It is very confusing too, because the two subclasses do different things. Does someone have to subclass the bot in a special way to be able to define `start`?

Reply:
Additionally, the codeblocks should be optional. Not everyone (especially those who this guide is targeted to) can read code very fluently so we should explain what is happening and what they should divert their attention towards.